config/jobs/kubernetes/kops/kops-presubmits-network-plugins.yaml

```diff
@@ -398,8 +404,8 @@ presubmits:
     - master
     run_if_changed: '^(upup\/models\/cloudup\/resources\/addons\/networking\.cilium\.io\/|pkg\/model\/(components\/containerd|firewall|components\/cilium|iam\/iam_builder)\.go|nodeup\/pkg\/model\/(context|networking\/cilium)\.go)'
     always_run: false
+    optional: false
     skip_report: false
-    optional: true
```

Comment:
might be a good time to permanently set this job to optional so we dont have to keep working around the drift in what build_jobs.py generates

Reply:
👍